lib/preload.js

```diff
@@ -1,3 +1,7 @@
 window.__nightmare = {};
 __nightmare.ipc = require('ipc');
 __nightmare.sliced = require('sliced');
+
+window.onerror = function (errorMsg, url, lineNumber, columnNumber, error) {
```

Comment:
probably want to do `window.addEventListener('error', fn)`

otherwise the page may overwrite this.